packages/augur-ui/src/modules/app/components/inner-nav/inner-nav.styles.less

```diff
@@ -185,7 +185,7 @@
 
   .InnerNav__menu--main,
   .InnerNav__menu--submenu {
-    height: 100vh;
+    height: calc(100vh - 100px);
```

Comment:
Is this for the ios footer issue?

Reply:
no. the page was too tall. because it wasn't accounting for the top banner